src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
Not sure I follow -- how is it one way?

This is what we have right now for :

Automatic: `let frameworkComponent = MyComponent.framework;`
Manual: `let frameworkComponent = MyComponent.driver('framework', framework);`

If we add in `driverResults` don't we end up with 3 different ways?

Automatic: `let frameworkComponent = MyComponent.framework;`
Automatic 2: `let frameworkComponent = MyComponent.driverResults.framework;`
Manual: `let frameworkComponent = MyComponent.driver('framework', framework);`